### DISCUSSION TITLE
Hearing others, but not me

### DISCUSSION THREAD

**Discussion by redwood75** [jamulussoftware/jamulus#3028] [Technical Support & Help]
I am new to Jamulus. Perhaps, it is a simple problem, but I can join and hear other people making music, but not me and my instrument.
I use Windows and a Focusrite interface (with Focusrite Asio USB driver).
As DAW, I use Reaper. Looking at the Jamulus volume slider, I can see no level at my track, so I think there have to be a problem between getting sound from Reaper/DAW to Jamulus.
Some help would be fine.

**Comment by chrisrimple** [comment]
In Jamulus, if you choose IN:Focusrite as your source, do you see your Jamulus input LEDs lighting up?  If so, then there is sound getting to the Focusrite but not to Reaper, and you should check the Reaper input settings.

**Reply by redwood75** [reply]
No, it is the opposite. I can hear a sound (produced within Reaper), but the Jamulus input LEDs are not lighting up. Output in Reaper and Jamulus are the same Focusrite channels.

**Reply by dcorson-ticino-com** [reply]
I would start out using only Jamulus and the Focusrite without the DAW.
When you have that working, add any other processing.
Remember that you need to be connected to a server in order to hear anything.

**Reply by redwood75** [reply]
Yes, this works (only Focusrite and Jamulus). There was an idea to use Rea Route within Reaper; I tried to follow these steps: https://docs.google.com/document/d/1tENfNKTWHasuTg33OdLLEo4-OOhWJolOo42ffSARxhY/edit
I got it to hear a sound, but it is awful and distorted, rather a computer music sound than the instrument sound. Do you have an idea what could be wrong?

**Reply by pljones** [reply]
Those instructions look fine.  Double check all your settings would be the first thing, if you've not already done so, of course.

**Reply by redwood75** [reply]
Now, it works. This video was also helpful: https://www.youtube.com/watch?v=6BkHVbsTkq0&t=655s
Thanks for your help!
